### PR TITLE
Update e2e framework for ILB and add basic test

### PR DESCRIPTION
**Pull request by spencerhance**
Adds HasAlphaRegionResource() to handle L7-ILB and adds it as a feature.  Ideally we should switch e2e to use composites and ResourceVersions.

The actual tests will be added in a seperate PR.

### CODE REVIEW

File: pkg/fuzz/features/ilb.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+package features
+
+import (
+	"k8s.io/api/networking/v1beta1"
+	"k8s.io/ingress-gce/pkg/fuzz"
+	"net/http"
+)
+
+// BackendConfigExample is an example of how a Feature will integrate with the
```
Comment (rramkumar1) [inline pkg/fuzz/features/ilb.go]: Fix this comment

File: cmd/e2e-test/ilb_test.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	"k8s.io/api/networking/v1beta1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/ingress-gce/pkg/annotations"
+	"k8s.io/ingress-gce/pkg/e2e"
+	"k8s.io/ingress-gce/pkg/fuzz"
+	"k8s.io/ingress-gce/pkg/fuzz/features"
+	"testing"
+)
+
+func TestILBBasic(t *testing.T) {
```
Comment (rramkumar1) [inline cmd/e2e-test/ilb_test.go]: This can be a separate PR but you probably want transition tests as well.

File: pkg/fuzz/features/ilb.go (hunk 1 above)
Comment (spencerhance) [reply pkg/fuzz/features/ilb.go]: Done

File: cmd/e2e-test/ilb_test.go (hunk 2 above)
Comment (spencerhance) [reply cmd/e2e-test/ilb_test.go]: Sounds good, there will be some follow up PRs with a whole bunch of tests

**Comment by spencerhance** [comment]
/hold
I think i'm missing some logic here

**Comment by bowei** [comment]
this mostly looks ok.

**Comment by spencerhance** [comment]
closing in favor of https://github.com/kubernetes/ingress-gce/pull/843
